config/environments/development.rb

```diff
@@ -7,6 +7,8 @@
 
   # Don't care if the mailer can't send
   config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.default_url_options = { host: '192.168.33.12:3000' }
```

Comment:
Unless you have a specific reason for doing so I would suggest this type of address be `localhost:3000` instead of your specific ip address. This way anyone who pulls down your code can work on it seamlessly from their machine. If you have some specific reason for doing it this way I wouldn't worry about it now. Just remember that it is best for this code to be as generic as possible and to keep specific data on your host machine.